### DISCUSSION TITLE
Investigate a software procurement standard

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/sci#233] [General]
From the conversations related to the SCI I realized that there is a tension between creating a standard for developers and creating a procurement standard for IT decision-makers. I believe the two should be distinct and we should explore creating a separate procurement standard. 

If you are going to spend money to support your software/business, either through purchasing software/services/platforms, what are the procurement rules that should be followed? What should you look for in organizations that provide this service? 

One thing you might look for is have they calculated an SCI score for their product or service?
Another one might be, have they set corporate sustainability targets, with dates, and have had their progress towards those targets verified by a 3rd party?
What is their policy regarding offsets and renewable energy? What does a good policy regarding offset purchasing and renewable energy look like? 

Most (I would argue 99%) of engineers work on large products where the purchasing choices have been decided, the platforms, hardware, services are fixed and they have to work the best they can within those constraints to build the most sustainable software. The SCI is a great metric for them to index against.

For the strata of engineers and IT decision-makers that do have influence over purchasing decisions, this software procurement standard can be for them.

**Comment by atg-abhishek** [comment]
@jawache - perhaps an extension to the SCI so that it all continues to tie together? With a wholly separate standard, there is a risk that they might drift? Another potential thing to consider here is that this could tie in even more closely with the reporting standard that we're developing which is the instrument based on which procurement decisions are made?

**Comment by Henry-WattTime** [comment]
Asim, this is interesting. I agree, I think a developer tool is very different than a comparative tool (for IT decisions makers). You've highlighted some interesting questions, but I think one of the largest struggles will be ensuring a consistent boundaries between different scores so that they are comparable.

Are we going to setout rules for how to assess the carbon commitments of different organizations? Is there a third party resource we can reference that is comprehensive? 

I found this report...very interesting:
https://newclimate.org/2022/02/07/press-release-corporate-climate-responsibility-monitor-2022/

**Comment by Sealjay** [comment]
I do like the SCI - and I can see a related standard being able to tie a procurement score to an SCI score. Maybe building a score bottom-up. I thought that could relate to the reporting project - ultimately becoming a competitive differentiator, picking the product with the lowest reported score.

**Reply by Sealjay** [reply]
Below that, you already have GRI scores: https://www.globalreporting.org/standards/media/2458/gri_standards_brochure.pdf

**Comment by Henry-WattTime** [comment]
Working group:
-Gadhu: people procuring have different priorities than engineers. SCI is very useful for engineers. Procurment decisions are different.
-Asim: people looking for simple advice
--do they have a SCI score
--have they trained
-Chris: have to live with procurement decisions in org, but consultants recommend services and structure
--procurement decisions are available
--would use scores to make decisions
--SCI is too granular for current decision making at the procurement level

SCI score is currently not useful for apples to apples comparison
Mitigation/offsets are important, but not part of the score
Corporate responsibility is important: https://greensoftware.foundation/articles/cta-success-factors-measurements-and-driving-the-right-behaviour

Much simpler mechanism for making procurement decisions?
-set of questions/checklists that you should be asking 
-GSF takes a position to drive behavior?
-becomes a useful tool for people procuring thing, could be added to an RFP

Todo: come up with potential set of questions that should be asked as part of procurement.
Maybe turn into specification/standard/model RFP language.

**Comment by atg-abhishek** [comment]
@jawache @Henry-WattTime I really like the idea of _model RFP language_ because that will be directly actionable with the procurement process and in alignment with the action-oriented approach that we take on in the SWG.
